crates/recent_projects/src/recent_projects.rs

```diff
@@ -364,6 +379,23 @@ impl PickerDelegate for RecentProjectsDelegate {
                 }),
         )
     }
+
+    fn render_footer(&self, cx: &mut ViewContext<Picker<Self>>) -> Option<AnyElement> {
+        if self.popover {
+            return None;
```

Comment:
This looks odd: 
* steals more vertical space
* renders the pointer mouse button on the footer albeit it's not clickable
* makes things even more different documentation-wise (one modal has docs in the tooltip, another in the footer now)